packages/contracts/.env.example

```diff
@@ -0,0 +1,11 @@
+# Name for the network to deploy to ("mainnet", "kovan", etc.)
+CONTRACTS_TARGET_NETWORK=
+
+# Private key that will send deployment transactions
+CONTRACTS_DEPLOYER_KEY=
+
+# RPC URL connected to the L1 chain we're deploying to
+CONTRACTS_RPC_URL=
+
+# Your Etherscan API key for the L1 network
+ETHERSCAN_API_KEY=
```

Comment:
After a quick text search, I don't see any of these 3 values used anywhere outside the deleted bash scripts.

Can these be deleted? Where would these values now be specified? I don't see where that would be done in the deploy configs.

Reply:
Ah, looks like they're already in hardhat.config.ts